src/main/java/org/openstreetmap/atlas/checks/utility/CommonMethods.java

```diff
@@ -19,10 +20,8 @@
      */
     public static long getOSMRelationMemberSize(final Relation relation)
     {
-        return relation
-                .members().stream().map(RelationMember::getEntity).map(entity -> entity.getType()
-                        .toString().concat(String.valueOf(entity.getOsmIdentifier())))
-                .distinct().count();
+        return relation.members().stream().map(RelationMember::getEntity)
```

Comment:
Although unlikely, we might run into a case where 2 different feature types with the same OSM ID exist in the same relation. To avoid returning 1 in such cases, we could do something similar to the previous method but every time we see a "node" or "point" we convert it to the same string, then attach the OSM id

Reply:
@seancoulter, i'm not sure if its possible in original OSM. every element in OSM has unique and permanent ID. do you know any examples ? I ran a query in our local OSM replication and couldn't find any.

Reply:
Here's a node and way with the same osm id. IDs should be unique by osm feature, but not necessarily across features: 
https://www.openstreetmap.org/way/152653228
https://www.openstreetmap.org/node/152653228

Since these aren't in the same relation there shouldn't be a problem in this particular example, but my comment was just pointing out the small probability that we'd encounter something like this in the same relation.

Reply:
thanks for the example! could we simply remove "POINT" type from the counting ? In my understanding all "POINT" type are artificial and added during the atlas generation. something like `.filter(entity -> !entity.getType().toString().equals("POINT"))`

Reply:
I think actually the Nodes are the ones added later on, and Points are present after raw atlas generation (Points generally being the Atlas analogue to OSM Nodes). Atlas Nodes would be added during way sectioning at specific locations (see https://github.com/osmlab/atlas/tree/dev/src/main/java/org/openstreetmap/atlas/geography/atlas/raw/sectioning#detailed-steps step 2) . So, we might not want to remove points from the count as they might not have a corresponding node -- but I think we can get away with removing either a point or node from the count if they both exist and have the same OSM identifier.

Reply:
@seancoulter, thanks for the feedback. should be good now.